chainio/clients/elcontracts/reader.go

```diff
@@ -116,7 +119,34 @@ func BuildELChainReader(
 	), nil
 }
 
+func BuildFromConfig(
```

Comment:
Should this just be called `NewFromConfig`? What's the rationale for using Build again?

Reply:
I don't have a strong pref. not sure why we used build in first place - maybe because it's building the bindings?

Reply:
I have a better names now - b9ba1bed85f61239f75c1c5b3d917c30ba4a9553
```
elcontracts.NewReaderFromConfig
```

I will add similar in future
```
elcontracts.NewWriterFromConfig
avsregistry.NewReaderFromConfig
avsregistry.NewWrtierFromConfig
```

wdyt?